test/fakeintake/server/server.go

```diff
@@ -368,7 +379,13 @@ func (fi *Server) handleDatadogRequest(w http.ResponseWriter, req *http.Request)
 }
 
 func (fi *Server) forwardRequestToDDDev(req *http.Request, payload []byte) error {
-	url := fi.forwardEndpoint + req.URL.Path
+	forwardEndpoint := fi.forwardEndpoint
+
+	if req.URL.Path == "/api/v2/logs" {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
v1 endpoint is similar, see https://docs.datadoghq.com/api/latest/logs/
```suggestion
	if req.URL.Path == "/api/v2/logs" || req.URL.Path == "/v1/input" {
```